site/_posts/2024-12-25-rustport.md

```diff
@@ -0,0 +1,316 @@
+---
+layout: post
+title: "Fish 4.0: The Fish Of Theseus"
```

Comment:
add a footnote to explain the allegory?

Reply:
I think it's well-known enough and it shows up later as well. People can google.

Reply:
Added a link later.